File: src/main/java/cc/turtl/cobbleaid/util/MiscUtils.java
package cc.turtl.cobbleaid.util;

import cc.turtl.cobbleaid.CobbleAid;
import net.minecraft.resources.ResourceLocation;

public class MiscUtils {

    public static ResourceLocation modResource(String path) {
        return ResourceLocation.fromNamespaceAndPath(CobbleAid.MODID, path);
    }
}

File: common/src/main/kotlin/cc/turtl/chiselmon/core/util/MiscUtils.kt
package cc.turtl.chiselmon.core.util

import cc.turtl.chiselmon.BuildDetails
import net.minecraft.resources.ResourceLocation

/**
 * Creates a ResourceLocation based on the Chiselmon mod id.
 */
fun modResource(path: String): ResourceLocation {
    return ResourceLocation.fromNamespaceAndPath(BuildDetails.MOD_ID, path)
}